tests/test_app.py

```diff
@@ -275,3 +275,21 @@ def test_zip_p(self) -> None:
             with open(json_filename, "r") as json_file:
                 report = json.load(json_file)
                 assert len(report) == SAMPLES_POST_CRED_COUNT + 1
+
+    def test_dup_p(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            sample_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "samples", "test_dup")
+            json_filename = os.path.join(tmp_dir, "report.json")
+            proc = subprocess.Popen([
+                sys.executable, "-m", "credsweeper", "--log", "debug", "--path", sample_path, "--save-json",
```

Comment:
`--log` option looks like used for debugging. Could you please remove `--log` option?